@here/harp-mapview/lib/SphereHorizon.ts

```diff
@@ -367,18 +368,14 @@ export class SphereHorizon {
 
     private get hFovVertical(): number {
         if (this.m_hFovVertical === undefined) {
-            this.m_hFovVertical = MathUtils.degToRad(this.m_camera.fov / 2);
+            this.m_hFovVertical = THREE.MathUtils.degToRad(this.m_camera.fov / 2);
```

Comment:
I know you didn't add it in this PR, but what does the `h` in `m_hFovVertical` mean?

Reply:
ahh... it means `half` ?

Perhaps add a comment to make this a bit clearer... I was wondering what the divide by 2

Reply:
renamed